ot3_state_manager/ot3_state_manager/measurable_states.py

```diff
@@ -0,0 +1,23 @@
+"""Contains the different states for OT3 that can be monitored."""
+from dataclasses import dataclass
+
+from ot3_state_manager.ot3_state_manager.ot3_axis import OT3Axis
+
+
+@dataclass
+class Position:
+    """Represents position state."""
+
+    axis: OT3Axis
+    current_position: float = 0.0
+    commanded_position: float = 0.0
+    encoder_position: float = 0.0
+
+
+@dataclass
+class Force:
+    """Represents force state."""
+
```

Comment:
could reasonably have an axis in here too

Reply:
c588d69673f561cba5773dff8239cf1593298f69
Is this correct @sfoster1?

Reply:
yeah